config/routes.rb

```diff
@@ -10,25 +10,14 @@
       end
     end
 
-    resources :products, :only => [] do
-      member do
-        get :available_repositories
-        put :toggle_repository
-      end
-    end
-
-    resources :providers, :only => [] do
-      collection do
-        get :redhat_provider
```

Comment:
There's a path helper referring to this route that should be updated unless that page is also dead code:

https://github.com/Katello/katello/blob/ffe498835c7ee2c6aa9795aee6292202453bb92f/app/views/katello/sync_management/_products.html.erb#L34

Reply:
Good catch, will fix.